templates/spring-boot-maven-mysql/files/README.md

```diff
@@ -1,14 +1,22 @@
-# Spring Boot Maven template for Platform.sh
+# Spring Boot Maven for Platform.sh
 
-This project provides a starter kit for Spring Boot Maven with MySQL projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Spring Boot Maven with MySQL projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+The Spring Framework is an application framework and inversion of control container for the Java platform. The framework's core features can be used by any Java application, but there are extensions for building web applications on top of the Jakarta EE platform. Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can "just run".
 
-## Platform.sh configuration files
+## Services
+
+* Java 8
+* mysql 10.2
```

Comment:
MariaDB

Reply:
Or at least properly capitalized (see previous).

Reply:
Done, thank you